yt_dlp/extractor/vlive.py

```diff
@@ -293,9 +295,9 @@ def _real_extract(self, url):
             entries, post_id, title, strip_or_none(post.get('plainBody')))
 
 
-class VLiveChannelIE(VLiveBaseIE):
+class VLiveChannelIE(VLiveIE):
     IE_NAME = 'vlive:channel'
-    _VALID_URL = r'https?://(?:channels\.vlive\.tv|(?:(?:www|m)\.)?vlive\.tv/channel)/(?P<id>[0-9A-Z]+)'
+    _VALID_URL = r'https?://(?:channels\.vlive\.tv|(?:(?:www|m)\.)?vlive\.tv/channel)/(?P<id>[0-9A-Z]+)(?:/board/[0-9]+)?'
```

Comment:
the valid url is matched with `re.match`. So looks like this change is unnecessary